force-app/main/default/lwc/datatablePicklistField/datatablePicklistField.html

```diff
@@ -0,0 +1,27 @@
+<template>
+    <div class={editClass}>
+        <div class="slds-grid slds-grid_align-spread">
+            <span class="slds-column slds-truncate">{displayValue}</span>
+            <template if:true={editable}>
+                <lightning-button-icon
+                    class="slds-column slds-m-left_x-small edit-button"
+                    icon-class="slds-icon-text-default" 
+                    variant="bare"
+                    icon-name="utility:edit" 
+                    onclick={handleEdit}
+                    >
+                </lightning-button-icon>
+            </template>
+        </div>
+    </div>
+    <div class="editor" if:true={editing}>
```

Comment:
There's nothing that hides the non-editable version of the element when it's being edited.